css-four-column-layout/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Four Column Layout</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <h1>How It Works</h1>
+    <main class="screens">
+      <div class="screen-1">
```

Comment:
make sure that you are using proper columns here. They need to have a set width / flex basis